### DISCUSSION TITLE
New Mevo+ ROI preventing reads

### DISCUSSION THREAD

**Discussion by hattglf** [springbok/MLM2PRO-GSPro-Connector#138] [Q&A] [answered]
Hi, having issues in new beta version v1.04.10 with the new added ROIs face to target, path, and AoA. Sometimes when doing short chips the Mevo+ will not read the club path (face to target, face to path), but it still reads everything else needed for GSPro to input the shot. However it seems that if the connector reads these fields as blank, it won't pass any data to GSPro.

Is there a way to configure the connector to still send the other fields to GSPro even if the club data is blank.

Thanks!

**Comment by wonder99** [comment]
Could you please post your log file from the appdata/logs directory?  I need to see the error message in this case.

**Reply by ThereIsNoBall** [reply]
Not the OP but I regularly experience the same issue. I played today so I grabbed some screenshots and logs. Hopefully they help. Let me know if you need more than the excerpts and cropped images. I do have the full versions. Thank you so much!
![FS Golf - Driver](https://github.com/user-attachments/assets/b8d09aba-3537-4460-8302-835b931af0d6)
![FS Golf - Wedge](https://github.com/user-attachments/assets/2e0a586c-c472-4060-960f-899f4e22e020)
[mlm2pro-gspro-connect - Driver.txt](https://github.com/user-attachments/files/17896706/mlm2pro-gspro-connect.-.Driver.txt)
[mlm2pro-gspro-connect - Wedge.txt](https://github.com/user-attachments/files/17896707/mlm2pro-gspro-connect.-.Wedge.txt)

**Reply by hattglf** [reply]
Yup, same issues I have and errors in the logs. Here are my logs.
[mlm2pro-gspro-connect.log](https://github.com/user-attachments/files/17905065/mlm2pro-gspro-connect.log)

**Comment by wonder99** [comment]
Thanks.  You can expect a release very soon with a proposed fix for these
MEVO+ issues.

**Reply by hattglf** [reply]
I tested the new release, and it appears the issue still exists in this version. It now shows up in the MLM2 App, but it does not send the shot to GSPro.
[mlm2pro-gspro-connect.log](https://github.com/user-attachments/files/17969596/mlm2pro-gspro-connect.log)


![image](https://github.com/user-attachments/assets/214cb4ab-ad64-426b-b4bd-539714c5b93d)
![image](https://github.com/user-attachments/assets/0f1ffe6a-a718-4ab6-bf08-e1acc3e11672)
![image](https://github.com/user-attachments/assets/144d5e42-65fb-4235-80e7-510db0931304)

**Comment by wonder99** [comment, marked as answer]
You are right, thanks for the files.  Please download this update and let
me know

https://www.dropbox.com/scl/fi/vy5kvkbzb7mtdxnozw3aj/MLM2Pro-GSPro-Connector_mevo.exe?rlkey=0hfel6kekpwptyu3puzwh1m9i&st=k94aoc8o&dl=1

- Paul

**Reply by hattglf** [reply]
That works, thank you!

**Reply by wittsjo** [reply]
@wonder99,  Your beta did not work for me. As the connector connects to the Mevo+ it jams/crashes so I can never get it running. Maybe I did something wrong when running it from another folder thatn the previously installed?Screenshots and log attached
[mlm2pro-gspro-connect.log](https://github.com/user-attachments/files/18143745/mlm2pro-gspro-connect.log)
![Screenshot 2024-12-13 182740](https://github.com/user-attachments/assets/d1f971fd-098f-440e-8735-bc79f9f597b0)

**Reply by wittsjo** [reply]
Nevermind, I got it working after some minor fixes on my side. It was and still is the short shots, like short chips that challenge the setup the most. It now processes it even if it doesn't have face patha nd club path etc. On the very short shots, I still loose some reads where it doesn't get the club speed. Does GSPro actually necessarily need club speed for simulating the shot?
![Screenshot 2024-12-15 215736](https://github.com/user-attachments/assets/bc29d81d-7538-46f2-bdea-6ad717d968bd)

**Comment by RocknRollGolf** [comment]
Hi

 I tried the new V1.04.16 
 And also had issues with the ROIs not reading . So I had to go back to 1.04.09

**Reply by wonder99** [reply]
Can you try the intermediate version here?
https://github.com/springbok/MLM2PRO-GSPro-Connector/discussions/138#discussioncomment-11429434

**Reply by RocknRollGolf** [reply]
![Screenshot 2024-12-15 123732](https://github.com/user-attachments/assets/ad08608d-ae0a-4a68-ab41-baa1a1ddb4ff)
Hi 

 ok so I had one miss read at first , also in /device it says not running, although it is working.

**Comment by wonder99** [comment]
No, club speed is just informational.

**Reply by wittsjo** [reply]
Then it would be great if that also did not prevent a shot "submission" to gspro.
